content/docs/code-splitting.md

```diff
@@ -44,7 +44,7 @@ console.log(add(16, 26)); // 42
 > 
 > Your bundles will end up looking a lot different than this.
 
-If you're using [Create React App](https://github.com/facebookincubator/create-react-app), [Next.js](https://github.com/zeit/next.js/), [Gatsby](https://www.gatsbyjs.org/), or a similar tool you will have a Webpack setup out of the box to bundle your
+If you're using [Create React App](https://github.com/facebookincubator/create-react-app), [Next.js](https://github.com/zeit/next.js/), [Gatsby](https://www.gatsbyjs.org/), or a similar tool, you will have a Webpack setup out of the box to bundle your
```

Comment:
Added an [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) because it can be clearer